public/javascripts/SVLabel/src/SVLabel/modal/ModalMission.js

```diff
@@ -63,6 +63,18 @@ function ModalMission (missionContainer, neighborhoodContainer, uiModalMission,
     };
 
     this._handleCloseButtonClick = function () {
+        mission = _missionContainer.getCurrentMission();
+        if(mission.getProperty("distanceProgress") < 0.0001) {
```

Comment:
I added this check because if a user was halfway through a mission and then came back to resume it later, it would log MissionStart even though they are already into the mission.

Reply:
Ahh makes sense!